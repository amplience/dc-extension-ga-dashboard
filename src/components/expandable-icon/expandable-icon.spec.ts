import { render } from '@testing-library/svelte';
import ExpandableIcon from './expandable-icon.svelte';

describe('ExpandableIcon', () => {
  it('should render the ExpandableIcon component in a collapsed state', () => {
    const { container } = render(ExpandableIcon, { expanded: false });

    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render the ExpandableIcon component in a expanded state', async () => {
    const { container } = render(ExpandableIcon, { expanded: true });

    expect(container.firstChild).toMatchSnapshot();
  });
});
