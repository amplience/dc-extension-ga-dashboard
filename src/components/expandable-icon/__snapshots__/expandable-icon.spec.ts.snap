// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpandableIcon should render the ExpandableIcon component in a collapsed state 1`] = `
<div>
  <div
    class="svelte-cy4end"
  >
    <div
      class=" svelte-zpbvvv"
      style="--icon-width: 24px; --icon-height: 24px;"
    >
      <object
        src="ic-expandable.svg/"
      />
    </div>
  </div>
</div>
`;

exports[`ExpandableIcon should render the ExpandableIcon component in a expanded state 1`] = `
<div>
  <div
    class="svelte-cy4end"
  >
    <div
      class=" svelte-zpbvvv"
      style="--icon-width: 24px; --icon-height: 24px;"
    >
      <object
        src="ic-contractable.svg/"
      />
    </div>
  </div>
</div>
`;
